docs/configuration/cosmos-conf.rst

```diff
@@ -54,6 +54,23 @@ This page lists all available Airflow configurations that affect ``astronomer-co
     - Default: ``None``
     - Environment Variable: ``AIRFLOW__COSMOS__DBT_DOCS_CONN_ID``
 
+.. _enable_profile_cache:
+
+`enable_profile_cache`_:
+    Enable caching for the DBT profile.
+
+    - Default: ``False``
+    - Environment Variable: ``AIRFLOW__COSMOS__ENABLE_PROFILE_CACHING``
+
+.. _profile_cache_path:
+
+`profile_cache_dir`_:
+    Folder path to store the DBT cached profile and related metadata
+
+    - Default: ``profile``
+    - Environment Variable: ``AIRFLOW__COSMOS__PROFILE_CACHE_DIR``
```

Comment:
I believe we should not have this setting. Why don't we create a subdirectory in the existing `cache_dir` and avoid forcing users to have to manually add another configuration?

Reply:
This is optional and we are creating a sub dir default name  `profile` in `cache_dir` see https://github.com/astronomer/astronomer-cosmos/pull/1046/files#diff-16ea4e2752796e174ffdf472c3bd2ca64e4e7c62ab586e169f2631fc9bb7e564R182